lib/logger.js

```diff
@@ -1,5 +1,5 @@
 module.exports = function logger(logLevel) {
-  var level = 2;
+  let level = 2;
```

Comment:
This could be called `defaultLevel` instead

Reply:
Now that I'm looking at the code again, it seems like the `default` branch of the `switch` right below provides the default value. I think I'll keep the variable name and leave it unassigned in this declaration.